### ISSUE TITLE
Error when input define as identity

### ISSUE THREAD

**Issue by hksonngan** [NVIDIA/triton-inference-server#1285]
**Description**
Loading servable: {name:  xxx version: 1} failed: Invalid argument: unable to load model 'xxx', configuration expects 1 inputs, model provides at most 0

**TRTIS Information**
What version of TRTIS are you using?
1.19.05
Are you using the TRTIS container or did you build it yourself?
TRTIS container

**To Reproduce**
Tensorflow 1.13.2
Input define as:
name: "input_x"
op: "Identity"
input: "padding_fifo_queue_Dequeue"
attr {
  key: "T"
  value {
    type: DT_FLOAT
  }
}
name: "softmax/Reshape_1"
op: "Reshape"
input: "softmax/Softmax"
input: "softmax/Shape"
attr {
  key: "T"
  value {
    type: DT_FLOAT
  }
}
attr {
  key: "Tshape"
  value {
    type: DT_INT32
  }
}
config.pbtxt
name: "lung_segmentation"
platform: "tensorflow_graphdef"
max_batch_size: 1
input [
  {
    name: "input_x"
    data_type: TYPE_FP32
    dims: [ -1 ]
  }
]
output [
  {
    name: "softmax/Reshape_1"
    data_type: TYPE_FP32
    dims: [ 256, 256, 2 ]
  }
]
default_model_filename: "lung_segmentation.pb"
**Expected behavior**
Success feeding input as tensorflow.

**Comment by deadeyegoodwin** [comment]
Triton doesn't have a version 1.19.05. Perhaps you mean 19.05? That is a very old release. You should try loading your model with the latest 20.03 release.

**Comment by hksonngan** [comment]
yes 19.05. Which is version of Tensorflow in TRTIS 20.03 container?

**Comment by deadeyegoodwin** [comment]
TF 1.15.0
See release notes: https://docs.nvidia.com/deeplearning/sdk/inference-release-notes/rel_20-03.html#rel_20-03
